angular-poller.js

```diff
@@ -72,6 +77,36 @@
                     poller.reset();
                 });
             }
+
+            /**
+             * delayAll/resetDelay or stopAll/startAll on visibilitychange.
+             */
+            if (pollerConfig.delayOnVisibilityChange) {
+                var handleDelayOnVisibilityChange = function () {
+                    if (isPageHidden()) {
+                        poller.delayAll();
+                    } else {
+                        poller.resetDelay();
+                    }
+                };
+
+                handleDelayOnVisibilityChange();
```

Comment:
From what I've tested in current use in my application, if you leave the tab before angular fully started, it won't catch `visibilitychange` event, so you are actually idle but never triggered stopAll/delayAll. So I just check right away for `document.hidden`